### DISCUSSION TITLE
Adding a `first-timers-only` issue label

### DISCUSSION THREAD

**Discussion by metanerd** [mattermost/community#119] [Practical Ideas]
https://github.com/MunGell/awesome-for-beginners  
We are already in there! But maybe the naming can be improved and make it really clear those should be picked up by first PR contributors, if it is not time critical. To keep them reserved for first timer contributors. 
https://github.com/search?utf8=%E2%9C%93&q=label%3Afirst-timers-only+is%3Aissue+is%3Aopen&type=

**Comment by esethna** [comment]
I think we used to have a "good first contribution" label cc// @jasonblais

**Reply by jasonblais** [reply]
Yea we have `Good First Issue` labels, they were standard labels used in GitHub previously, but that may have changed - here are a few issues that have it: https://github.com/mattermost/mattermost-server/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22

**Reply by metanerd** [reply]
OK, so maybe instead of Good First Issue/first-timers-only we could set up some monitoring to have Good First Issue and Up For Grabs in our main repos available to be picked up.
